packages/panna-sdk/src/react/components/connect-button.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
+
+import { useDialog } from '@/hooks/use-dialog';
+import { AuthenticationFlow } from './authentication-flow';
+import { Button } from './ui/button';
+import { Dialog, DialogTrigger } from './ui/dialog';
+
+export function ConnectButton() {
```

Comment:
why a LoginButton and ConnectButton? its a bit confusing. Lets just have a single LoginButton

Reply:
The LoginButton houses the authenticated and unauthenticated components. So having the auth flow within the same component would complicate the component and make it harder to read.It also gives the component multiple responsibilities instead of a single one.